### ISSUE TITLE
Exception: builtins.KeyError: 'number'

### ISSUE THREAD

**Issue by wsanchez** [burningmantech/ranger-ims-server#171]
```
2017-08-07T21:31:09-0700 [ims.application._klein#critical] Request failed
        Traceback (most recent call last):
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 126, in execute_endpoint
            return endpoint_f(self._instance, *args, **kwargs)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 211, in _f
            return _call(instance, f, request, *a, **kw)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/klein/app.py", line 51, in _call
            result = ensureDeferred(result)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 908, in ensureDeferred
            return _inlineCallbacks(None, coro, Deferred())
        --- <exception caught here> ---
          File "/opt/ims/lib/env/lib/python3.6/site-packages/twisted/internet/defer.py", line 1386, in _inlineCallbacks
            result = g.send(result)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/application/_api.py", line 565, in newIncidentReportResource
            incidentReport = modelObjectFromJSONObject(json, IncidentReport)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/model/json/_json.py", line 61, in modelObjectFromJSONObject
            return jsonDeserialize(object, modelClass)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/cattr/converters.py", line 138, in structure
            if not isinstance(cl, _Union)
          File "/opt/ims/lib/env/lib/python3.6/site-packages/cattr/converters.py", line 130, in <lambda>
            self._structure.register(cl, lambda t, o: func(o, t))
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/model/json/_report.py", line 85, in deserializeIncidentReport
            for key in IncidentReportJSONKey
          File "/opt/ims/lib/env/lib/python3.6/site-packages/ims/model/json/_report.py", line 85, in <genexpr>
            for key in IncidentReportJSONKey
        builtins.KeyError: 'number'
```

**Comment by wsanchez** [comment]
I think this happens when the web client tries to create a new incident report, because the JS doesn't provide a number in the JSON, since the server has to assign one.

**Comment by wsanchez** [comment]
Fixed by #173
